.github/workflows/sycl-docs.yml

```diff
@@ -49,7 +49,13 @@ jobs:
         mkdir clang
         mv $GITHUB_WORKSPACE/build/tools/sycl/doc/html/* .
         mv $GITHUB_WORKSPACE/build/tools/clang/docs/html/* clang/
+        cp -r $GITHUB_WORKSPACE/repo/devops/scripts/benchmarks/html benchmarks
         touch .nojekyll
+        # Update benchmarking dashboard configuration
+        cat << 'EOF' > benchmarks/config.js
+        remoteDataUrl = 'https://raw.githubusercontent.com/intel/llvm-ci-perf-results/refs/heads/unify-ci/UR_DNP_INTEL_06_03/data.json';
```

Comment:
This really shouldn't be in a directory keyed by the system name. The data.json can contain results from multiple systems.  That's how it works here: https://github.com/oneapi-src/unified-runtime/tree/benchmark-results where we store results from both BMG and PVC systems.